docs/interactions/Slash_Commands.md

```diff
@@ -790,21 +792,24 @@ If you specify `choices` for an option, they are the **only** valid values for a
-|------------|------------------------------------------------------------------|----------------------------------------------------------------|
-| id         | snowflake                                                        | id of the interaction                                          |
-| type       | InteractionType                                                  | the type of interaction                                        |
-| data?\*    | ApplicationCommandInteractionData                                | the command data payload                                       |
-| guild_id   | snowflake                                                        | the guild it was sent from                                     |
-| channel_id | snowflake                                                        | the channel it was sent from                                   |
-| member     | [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) object | guild member data for the invoking user, including permissions |
-| token      | string                                                           | a continuation token for responding to the interaction         |
-| version    | int                                                              | read-only property, always `1`                                 |
+| Field        | Type                                                             | Description                                                    |
+|--------------|------------------------------------------------------------------|----------------------------------------------------------------|
+| id           | snowflake                                                        | id of the interaction                                          |
+| type         | InteractionType                                                  | the type of interaction                                        |
+| data?\*      | ApplicationCommandInteractionData                                | the command data payload                                       |
+| guild_id?    | snowflake                                                        | the guild it was sent from                                     |
+| channel_id?  | snowflake                                                        | the channel it was sent from                                   |
+| member?\*\*  | [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) object | guild member data for the invoking user, including permissions |
+| user?        | [user](#DOCS_RESOURCES_USER/user-object) object                  | user object for the invoking user, if invoked in a DM          |
+| token        | string                                                           | a continuation token for responding to the interaction         |
+| version      | int                                                              | read-only property, always `1`                                 |
```

Comment:
Should the version be bumped to `2` cause this is a breaking change (`guild_id` no longer has to always be there)? At the same time it's a beta so this is too be expected...

Reply:
We currently aren't planning on bumping this to `2` because it's....breaking ish? It is a breaking change specifically for commands that happen in DMs, but commands in DMs is not something that is supported until we roll out this code. And this behavior does not change for guild commands (the interaction payload you receive from a guild today will still have all the same data).

We are still in a beta so we _can_ make breaking changes, but we also will coordinate with developers and maintainers more on rollouts in the future to have more explicit beta periods, so we are trying not to make explicitly breaking changes during this beta period and will save those for a future v2.

Reply:
> breaking ish

if not for the beta, this would absolutely warrant a bump. if people's code has to change (be updated to tolerate guild_id not being present), it is breaking. that it has to do with a new functionality (DMs) is irrelevant.